airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3410,6 +3412,32 @@ components:
           description: Set state of DAG runs to RUNNING.
           type: boolean
 
+        dag_run_id:
+          type: string
+          description:  The DagRun ID for this task instance
+          nullable: true
+
+        include_upstream:
+          description: If set to true, upstream tasks are also affected.
+          type: boolean
+          default: false
+
+        include_downstream:
+          description: If set to true, downstream tasks are also affected.
+          type: boolean
+          default: false
+
+        include_future:
+          description: If set to True, also tasks from future DAG Runs are affected.
+          type: boolean
+          default: false
+
+        include_past:
+          description: If set to True, also tasks from past DAG Runs are affected.
+          type: boolean
+          default: false
+
```

Comment:
@bbovenzi @ephraimbuddy I have added the params but when I hit these params to test the API, I get an error saying `Unknown field`. Have anyone faced this issue? Can you tell me where I am going wrong?

Reply:
@bbovenzi @ephraimbuddy Could anyone help me in solving this issue? It would be very helpful for me.

Reply:
Sorry, I am not too familiar with this side of things, but I don't seem to see `dry_run` in this file. Could that be it?

Reply:
Also you need to add these to the Marshmallow schema (you already added `dag_run_id` but not the others).